distribution/docker/peon.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
```

Comment:
Meta commentary: it would be cool (and I don't think too crazy) if there was a Cli target that is like "task" which essentially does a remote bootstrap.

I.e. it could start up and the first thing it does is read some config for the overlord process, "phone home" to ask for the task spec and runtime properties, and then use those to bootstrap.  

This might open up security holes, so it would probably have to be done with some sort of shared secret or something, maybe?  But anyway, might simplify this script into something that is relatively generic and not even k8s-dependent.

Could also be done as an evolution of this change post-merge.

Reply:
So when creating this patch, I was hoping to minimize core changes to increase the odds of this patch getting accepted.  In all honesty, I believe if this is a first class feature, we can fix some other hacky stuff that was done.  I definitely think having a Cli command for the task is a good idea, I think this is something that I could put up a PR if /when this gets merged.